CHANGELOG.rst

```diff
@@ -19,6 +19,12 @@ The format is based on `Keep a Changelog`_.
 `UNRELEASED`_
 =============
 
+Added
+-----
+
+- The option `vertical_compression` to the text drawer allows to control how much
+  room the text circuit drawing takes.
+
```

Comment:
We should mention that we added the parameter to the top level argument too

Reply:
Fixed in c290fc64